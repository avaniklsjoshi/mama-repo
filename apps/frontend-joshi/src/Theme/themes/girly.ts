export default {
  themeType: "girly",
  colors: {
    background: "#f1b7ea",
    text: "#fff"
  },
  header: {
    background: "#000",
    text: "#fff"
  },
  footer: {
    background: "#262726",
    text: "#fff"
  },
  section: {
    background: "#008F7A",
    text: "#fff"
  }
};
