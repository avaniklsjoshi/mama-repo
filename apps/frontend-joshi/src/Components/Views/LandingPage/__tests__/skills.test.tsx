/* eslint-disable testing-library/no-node-access */
/* eslint-disable testing-library/no-container */
import { act, fireEvent, render, screen } from "@testing-library/react";
import { ThemeProvider } from "styled-components";
import girly from "../../../../Theme/themes/girly";
import Skills from "../skills";

jest.mock("react-i18next", () => ({
  useTranslation: () => ({ t: (key: any) => key })
}));

describe("Landing Page", () => {
  it("renders Skills", () => {
    const { container } = render(
      <ThemeProvider theme={girly}>
        <Skills />
      </ThemeProvider>
    );
    expect(container).toMatchSnapshot();
    expect(screen.getByText("techStack")).toBeInTheDocument();
    expect(screen.getByText("tools")).toBeInTheDocument();
    expect(screen.getByText("supporting")).toBeInTheDocument();
  });
});
