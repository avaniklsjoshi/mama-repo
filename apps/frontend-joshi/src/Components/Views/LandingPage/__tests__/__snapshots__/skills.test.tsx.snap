// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Page renders Skills 1`] = `
<div>
  <div
    class="skills-container"
  >
    <div
      class="sc-bczRLJ jliZnU skills-title"
    >
      skills
    </div>
    <div
      class="container"
    >
      <div
        class="col"
      >
        <h1
          class="titulo"
        >
          techStack
        </h1>
        <ul
          class="skill"
        >
          <li>
            <span
              class="blueline javascript"
            />
            <em>
              Javascript
            </em>
          </li>
          <li>
            <span
              class="blueline reactJS"
            />
            <em>
              React JS
            </em>
          </li>
          <li>
            <span
              class="blueline nodeJS"
            />
            <em>
              NodeJS
            </em>
          </li>
          <li>
            <span
              class="blueline html"
            />
            <em>
              HTML
            </em>
          </li>
          <li>
            <span
              class="blueline cssScss"
            />
            <em>
              CSS & Scss
            </em>
          </li>
          <li>
            <span
              class="blueline mongoDB"
            />
            <em>
              MongoDB
            </em>
          </li>
          <li>
            <span
              class="blueline docker"
            />
            <em>
              Docker
            </em>
          </li>
        </ul>
      </div>
      <div
        class="col"
      >
        <h1
          class="supporting titulo"
        >
          supporting
        </h1>
        <ul
          class="skill"
        >
          <li>
            <span
              class="greenline typescript"
            />
            <em>
              Typescript
            </em>
          </li>
          <li>
            <span
              class="greenline reduxSaga"
            />
            <em>
              Redux, Saga
            </em>
          </li>
          <li>
            <span
              class="greenline styledComponents"
            />
            <em>
              Styled Components
            </em>
          </li>
          <li>
            <span
              class="greenline webpackVite"
            />
            <em>
              Webpack, Vite
            </em>
          </li>
          <li>
            <span
              class="greenline estsLint"
            />
            <em>
              EsLint, TsLint
            </em>
          </li>
          <li>
            <span
              class="greenline expressJS"
            />
            <em>
              Express JS
            </em>
          </li>
          <li>
            <span
              class="greenline jestTesting"
            />
            <em>
              Unit Testing
            </em>
          </li>
          <li>
            <span
              class="greenline storybook"
            />
            <em>
              StoryBook
            </em>
          </li>
        </ul>
      </div>
      <div
        class="col"
      >
        <h1
          class="tools titulo"
        >
          tools
        </h1>
        <ul
          class="skill"
        >
          <li>
            <span
              class="redline vscode"
            />
            <em>
              Visual Studio code
            </em>
          </li>
          <li>
            <span
              class="redline compass"
            />
            <em>
              MongoDB Compass
            </em>
          </li>
          <li>
            <span
              class="redline sourcetree"
            />
            <em>
              Source Tree
            </em>
          </li>
          <li>
            <span
              class="redline postman"
            />
            <em>
              Postman
            </em>
          </li>
          <li>
            <span
              class="redline herokuMlab"
            />
            <em>
              Heroku, MLab
            </em>
          </li>
          <li>
            <span
              class="redline xcode"
            />
            <em>
              XCode
            </em>
          </li>
          <li>
            <span
              class="redline browsers"
            />
            <em>
              Browsers
            </em>
          </li>
        </ul>
      </div>
    </div>
  </div>
</div>
`;
